rockcraft.yaml

```diff
@@ -116,6 +116,8 @@ parts:
             chown -R 584792 django/app/reports
             chown -R 584792 django/app/scripts
     django-framework/dependencies:
+        overlay-packages:
+            - ca-certificates
```

Comment:
Is this change intentional? It is not a doc change.

Reply:
I found the issue while testing git data sources (following the docs in this PR). I added it here for convenience (it is reflected in the PR info)